src/client/Client.cc

```diff
@@ -9769,18 +9787,20 @@ int64_t Client::_preadv_pwritev_locked(Fh *fh, const struct iovec *iov,
         if (r <= 0)
           return r;
 
+        client_lock.unlock();
```

Comment:
What would be more conventional here would be to pass the `std::unique_lock<>&` to these helper methods so they can manipulate the lock that way.